### ISSUE TITLE
Fails with Error

### ISSUE THREAD

**Issue by MustardLighthouse** [xxyzz/WordDumb#52]
### Describe the bug

When running the plugin on any book it will fail with similar errors.

### System Information

Mac OS 12.4, Calibre 6.1, Python 3

### Error message

```text
Starting job: Generating Word Wise and X-Ray for Becoming 
Job: "Generating Word Wise and X-Ray for Becoming" failed with error: 
Traceback (most recent call last):
  File "calibre/gui2/threaded_jobs.py", line 82, in start_work
  File "calibre_plugins.worddumb.parse_job", line 147, in do_job
    run_subprocess(args, input_str)
  File "calibre_plugins.worddumb.utils", line 48, in run_subprocess
    return subprocess.run(
  File "subprocess.py", line 524, in run
subprocess.CalledProcessError: Command '['/usr/bin/python3', '/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip', 'BBEX21IOI6', '/Users/samuel/OneDrive/Calibre Library/Michelle Obama/Becoming (133)/Becoming - Michelle Obama.mobi', 'Becoming', 'e6ccc3c0', 'en_core_web_md', 'en', 'utf-8', '3.23.1', 'cn', '', 'MOBI', '1', '/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip', '', '-l', '-x']' returned non-zero exit status 1.
 
Called with args: ((133, 'MOBI', '/Users/samuel/OneDrive/Calibre Library/Michelle Obama/Becoming (133)/Becoming - Michelle Obama.mobi', , {'spacy': 'en_core_web_', 'wiki': 'en', 'kaikki': 'English'}), True, True) {'notifications': , 'abort': , 'log': } 


Another one:

Starting job: Generating Word Wise and X-Ray for Becoming 
Job: "Generating Word Wise and X-Ray for Becoming" failed with error: 
Traceback (most recent call last):
  File "calibre/gui2/threaded_jobs.py", line 82, in start_work
  File "calibre_plugins.worddumb.parse_job", line 147, in do_job
    run_subprocess(args, input_str)
  File "calibre_plugins.worddumb.utils", line 48, in run_subprocess
    return subprocess.run(
  File "subprocess.py", line 524, in run
subprocess.CalledProcessError: Command '['/usr/bin/python3', '/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip', 'BBEX21IOI6', '/Users/samuel/OneDrive/Calibre Library/Michelle Obama/Becoming (133)/Becoming - Michelle Obama.mobi', 'Becoming', 'e6ccc3c0', 'en_core_web_md', 'en', 'utf-8', '3.23.1', 'cn', '', 'MOBI', '1', '/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip', '', '-l', '-x']' returned non-zero exit status 1.
 
Called with args: ((133, 'MOBI', '/Users/samuel/OneDrive/Calibre Library/Michelle Obama/Becoming (133)/Becoming - Michelle Obama.mobi', <calibre.ebooks.metadata.book.base.Metadata object at 0x124601990>, {'spacy': 'en_core_web_', 'wiki': 'en', 'kaikki': 'English'}), True, True) {'notifications': <queue.Queue object at 0x124601b40>, 'abort': <threading.Event object at 0x124601a50>, 'log': <calibre.utils.logging.GUILog object at 0x124601c00>} 
Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip/__main__.py", line 55, in <module>
  File "/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip/parse_job.py", line 322, in create_files
  File "/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip/x_ray.py", line 196, in finish
  File "/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip/x_ray.py", line 83, in insert_descriptions
  File "/Users/samuel/Library/Preferences/calibre/plugins/WordDumb.zip/mediawiki.py", line 257, in inception_text
ValueError: Invalid isoformat string: 'http://www.wikidata.org/.well-known/genid/7ea35d4504b835fc37b90f06a6e522a'
```

### Reproduce steps

Run word dumb

**Comment by xxyzz** [comment]
You didn't post the plugin version, this bug should have been fixed since v3.23.0.

**Comment by MustardLighthouse** [comment]
Hello, sorry for not specifying. I am on 3.23.1. 

I deleted ~/Library/Preferences/calibre/plugins/worddumb-wikimedia/wikidata.json and now it appears to be working. Thanks!
